.circleci/config.yml

```diff
@@ -110,5 +94,5 @@ workflows:
       - manylinux2014-aarch64:
           matrix:
             parameters:
-              NRN_PYTHON_VERSION: ["37", "38", "39", "310", "311"]
+              NRN_PYTHON_VERSION_MINOR: ["9", "10", "11", "12", "13"]
```

Comment:
What's our policy on supporting deprecated Python versions? I think we shouldn't drop support for a patch release, maybe for a minor one it'd be justified, but for a patch it seems a bit much since one could argue dropping support for a Python version is a breaking change.

On the other hand, I don't know if any of the CI providers we are currently using actually _have_ support for 3.7 and 3.8; if they don't, we would kind of be forced to either 1) drop support for them as well, or 2) build any missing releases manually. Here's to hoping that they are still supported in all relevant CIs so we can avoid either of the options above 😅